llama-index-integrations/llms/llama-index-llms-ipex-llm/pyproject.toml

```diff
@@ -33,7 +33,7 @@ readme = "README.md"
 version = "0.1.6"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4.0"
+python = ">=3.8.1,<4.0"
```

Comment:
I thiiink the actual issue, after looking at the logs, is the `<4.0`, while other packages have a more strict 3.12 dependency

Reply:
Ohhh...okay thanks for taking a look.

Reply:
I changed it to ">=3.9,3.12" now... wondering if i should just use the suggested ">=3.8.1,<3.12" lol

Reply:
I have a feeling ipex is only 3.9+ compatible. Let's see if this publishes nicely